apps/backend/tests/conftest.py

```
@@ -155,3 +166,98 @@ def redis_service(use_external_services):
+    )
+    
+    async with async_session() as session:
+        yield session
+
+
+@pytest.fixture
+def client(test_session):
+    """Create test client."""
+    def override_get_db():
+        return test_session
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)